examples/test.html

```diff
@@ -58,13 +58,18 @@
   attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
   }).addTo(map);
 
+  var layers = [{
+      sql: 'select * from ne_10m_populated_places_simple_3',
```

Comment:
cartodb deals with sql not with tables, all the client side apis have been changed in order to work only with sql. Table support is only for backwards compatibility